StortSpelProjekt/Engine/src/Engine.h

```diff
@@ -20,9 +20,10 @@
 // Sub-engines
 #include "Renderer/Renderer.h"
 #include "Renderer/Transform.h"
+#include "Renderer/Model.h"
 #include "Renderer/Mesh.h"
 #include "Renderer/BaseCamera.h"
-// #include "physics"
+#include "Physics/Physics.h"
```

Comment:
This should be forward declaration

Reply:
In engine..h? Nothing else is forward declared.